### PR TITLE
Feature/dark mode

### LINKED ISSUE #120: [Feature] Add Dark Mode
### Feature Description

- Add a button to enable dark mode styles in addition to the original light mode style

### Reason for Change

- Enable easier viewing at night for users when in dark mode

### Where are the changes needed?

App.tsx
Index.css

### Code Snippets

_No response_

### Anything Else?

_No response_

### PR DESCRIPTION
**Pull request by mwan391**
### Changes

- added a toggle button to enable dark mode or light mode

### Tests applied

Manual testing

Screenshot 2023-10-19 at 22 52 19

### Issue ticket number(s)

Closes #120 

### Checklist

- [ ] Documented
- [ ] Linting tests successful
- [ ] merge updated prod branch into development branch

### CODE REVIEW

File: pnpm-lock.yaml
Comment (rhun558) [inline pnpm-lock.yaml:1]: package.json and pnpm-lock.yaml i dont think should be included in the pr, as these are autogenerated afaik.

**Review by calebWei — APPROVED** [review]
Looks good to me!

File: pnpm-lock.yaml
Comment (mwan391) [reply pnpm-lock.yaml:1]: got it. i'll try run pnpm install to fetch any new changes

**Comment by mwan391** [comment]
Addition: Hover highlights are determined by light or dark mode now

- Highlight hover in dark mode
Screenshot 2023-10-20 at 00 04 35

- Highlight hover in light mode
Screenshot 2023-10-20 at 00 04 48

**Review by calebWei — APPROVED** [review]
Great job on the additional fixes / changes!

**Pull request merged**